microsoft-support.md

```diff
@@ -12,10 +12,9 @@ This table describes support type, supported patch version and end of support da
 
 |  Version  |  Release Date | Support Level | Supported Patch Version | End of Support |
 | -- | -- | -- | -- | -- |
-| [.NET Core 3.1](https://devblogs.microsoft.com/dotnet/announcing-net-core-3-1/) | December 3, 2019 | LTS | [3.1.0](https://dotnet.microsoft.com/download/dotnet-core/3.1) | |
-| [.NET Core 3.0](https://devblogs.microsoft.com/dotnet/announcing-net-core-3-0/) | September 23, 2019 | Current | [3.0.0](https://dotnet.microsoft.com/download/dotnet-core/3.0) | March 23, 2020 |
-| [.NET Core 2.2](https://aka.ms/netcore22announce) | December 04, 2018 | Current | [2.2.8](https://dotnet.microsoft.com/download/dotnet-core/2.2) | December 23, 2019 |
-| [.NET Core 2.1](https://blogs.msdn.microsoft.com/dotnet/2018/05/30/announcing-net-core-2-1) | May 30, 2018 | LTS | [2.1.14](https://dotnet.microsoft.com/download/dotnet-core/2.1) | At least three years from LTS declaration (August 21, 2018). |
+| [.NET Core 3.1](https://devblogs.microsoft.com/dotnet/announcing-net-core-3-1/) | December 3, 2019 | LTS | [3.1.0](https://dotnet.microsoft.com/download/dotnet-core/3.1) | December 3, 2022 |
```

Comment:
> Three years from LTS declaration ([August 21, 2018](https://devblogs.microsoft.com/dotnet/net-core-august-2018-update/)).

The text for 3.1 and 2.1 is different. We should probably just delete the text above and make it simpler.

Reply:
The only reason it was different is because the declaration for 2.1 LTS happened at a different date and there was a link to share for the same. I can still delete the text.